tests/containers/helm.pm

```diff
@@ -121,9 +125,31 @@ sub post_fail_hook {
     script_run("kubectl config set-context --current --namespace=default");
     script_run("kubectl delete namespace helm-ns-$job_id");
 
+    helm_cleanup() unless $self->{is_k3s};
     uninstall_k3s() if $self->{is_k3s};
 }
 
+sub helm_cleanup {
+    diag("helm_cleanup started.");
```

Comment:
AFAIK diag will only print something in os-autoinst log , maybe better to do record_info instead ? ( or in addition ? )

Reply:
I switched from `record_info` to `diag` to not clutter the test overview that much. those messages are anyways not necessary for the 'test' - they're just 'administrative' informations.